src/Ledger.Net.Tests/LedgerTests.cs

```diff
@@ -17,26 +17,40 @@ public class LedgerTests
 
         static readonly UsageSpecification[] _UsageSpecification = new[] { new UsageSpecification(0xffa0, 0x01) };
 
-        [Fact]
```

Comment:
I just commented out all Bitcoin related methods for the time being. Nothing was actually removed from this class.